langkit/regexes.py

```diff
@@ -38,17 +38,19 @@ def wrappee(text):
 _registered = False
 
 
-def _register_udfs():
+def _register_udfs(config: Optional[LangKitConfig] = None):
     global _registered
-    if _registered:
+    if _registered and config is None:
```

Comment:
The old name is not "unregistered," so you'll get both.

Reply:
good callout, fixing!